src/Test.UnitTests.Sarif.Multitool/FileWorkItemsCommandTests.cs

```diff
@@ -0,0 +1,217 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif.Multitool;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Multitool
+{
+    public class FileWorkItemsCommandTests
+    {
+        [Theory]
```

Comment:
>Theory [](start = 9, length = 6)

No [Theory] usage please! If you'll recall, this convention tanked unit test performance previously and I had to go to considerable effort to undo them and make tests run in a reasonable time. There are other patterns that work just fine and which don't provoke whatever reflection costs this mechanism appears to.

Reply:
Will do later. Not as important as getting bug filing working this week.

---
In reply to: [306578562](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306578562) [](ancestors = 306578562)